app/views/admin/users/edit.html.erb

```diff
@@ -34,7 +34,9 @@
     <%= form_for(@user, url: user_status_admin_user_path(@user), html: { class: "mt-3" }) do |f| %>
       <div class="form-group">
         <%= f.label "Select new user status", class: "mr-3" %>
-        <%= f.select(:user_status, options_for_select(["Warn", "Comment Ban", "Ban", "Regular Member", "Trusted", "Pro"]), include_blank: true) %>
+        <% options = { "Base Roles" => Constants::BASE_ROLES } %>
+        <% options["Special Roles"] = Constants::SPECIAL_ROLES if current_user.has_role?(:super_admin) %>
+        <%= f.select(:user_status, grouped_options_for_select(options), include_blank: true) %>
```

Comment:
I think if we could make these roles look prettier here that would be nice too. For example:
![Screen Shot 2020-08-18 at 10 52 24 AM](https://user-images.githubusercontent.com/17884966/90528696-f010a280-e140-11ea-9db5-9f47b79162af.png)

The "Current Roles" don't distinguish which `single_resource_admin` roles they are. That said they do in `/admin/permissions` but I feel like it's more helpful in this single user view.